packages/foam-vscode/package.json

```diff
@@ -166,6 +166,10 @@
         "command": "foam-vscode.open-daily-note",
         "title": "Foam: Open Daily Note"
       },
+      {
+        "command": "foam-vscode.open-daily-note-for-date",
```

Comment:
instead of adding a new command, could we retrofit/expand the previous `foam-vscode.open-daily-note` command to default to today (as it currently is), but to possibly also take a parameter with the date to open?

Reply:
How would we pass a parameter to a command? Do you have an example for me?

Reply:
You might be able to get an idea from `utility-commands.ts` and https://stackoverflow.com/questions/34809864/arguments-for-commands-in-vs-code

does that help?